framework/web/MultipartFormDataParser.php

```diff
@@ -0,0 +1,357 @@
+<?php
+/**
+ * @link http://www.yiiframework.com/
+ * @copyright Copyright (c) 2008 Yii Software LLC
+ * @license http://www.yiiframework.com/license/
+ */
+
+namespace yii\web;
+
+use yii\base\Object;
+use yii\helpers\ArrayHelper;
+use yii\helpers\StringHelper;
+
+/**
+ * MultipartFormDataParser parses content encoded as 'multipart/form-data'.
+ * This parser provides the fallback for the 'multipart/form-data' processing on non POST requests,
+ * for example: the one with 'PUT' request method.
```

Comment:
Just to be consistent

https://github.com/yiisoft/yii2/blob/master/framework/widgets/ActiveForm.php#L33
https://github.com/yiisoft/yii2/blob/master/framework/filters/AccessRule.php#L60